markdown/developers/adding_pipelines.md

```diff
@@ -87,7 +87,7 @@ The repository should already contain the required `.travis.yml` file, so the ne
 time you push a commit the tests will be automatically triggered.
 
 The process for Docker Hub is similar, though a little more fiddly. Docker has a number
-of  different docker websites (docker hub, docker cloud, docker swarm _etc._), but
+of  different docker websites (Docker Hub, Docker Cloud, Docker Swarm _etc._), but
```

Comment:
Is this still valid? Does Docker Cloud / Swarm still exist?

Reply:
They still exist, but the current docs say we should always use Docker Hub. (so could drop all the others)

Reply:
Yes, it also felt a bit outdated to me, so I removed the whole paragraph now.